### ISSUE TITLE
I have issue with jio cinema and zee5 

### ISSUE THREAD

**Issue by SaiKiranCruzz** [Vikassm73/AjaykRepo#241]
![16961487300584100980317152117450](https://github.com/Vikassm73/AjaykRepo/assets/84628918/4a9ac91b-a408-4fd2-82d7-d4267660925c)
![16961487648031806008918366182244](https://github.com/Vikassm73/AjaykRepo/assets/84628918/6b7c49ac-8954-4f77-a9f7-f5befe381bb8)

**Comment by SaiKiranCruzz** [comment]
@Vikassm73 sir pls look into this since 3-4days im having this issue how can i fix this?

**Comment by Vikassm73** [comment]
Both the version are working fine. Are you using VPN. Those will not work under VPN. reinstall app again and try otherwise send me a kodi debug log if problem persist.

**Comment by SaiKiranCruzz** [comment]
Hey hello jio cinema got fixed after the update but zee5 is still the same. Shows same error as shown in previous screen shoot and
 here is the log https://paste.kodi.tv/gujititinu
 And before aslo i wasn't able to login through otp coz there was password option instead of otp but i cannot login through password coz right now there is no password option is available on zee5 website n app the password option got replaced with otp. there is no option on add-on settings of see 5 to get an otp. And i dont use any vpn ty
![16963237036542863209432886412518](https://github.com/Vikassm73/AjaykRepo/assets/84628918/c1f63f90-9bc8-4ebf-84df-338813f8f97c)

**Comment by Vikassm73** [comment]
Just enter your mobile number in the setting. Keep password blank.

**Comment by SaiKiranCruzz** [comment]
Helo i did what u said but still same issue its not even working on kodi 20 tried both versions on kodi 19.5 n 20 also still same error in screenshot tried it on mobile phone also if i had some issue on tv but on mobile phone also same issue addon has some issue i guess coz a week ago there was no such error
![Uploading Screenshot_2023-10-03-15-32-44-57_e18329511ee9ae4541720d804dbbeba9.jpg…]()

**Comment by SaiKiranCruzz** [comment]
![Uploading Screenshot_2023-10-03-15-32-44-57_e18329511ee9ae4541720d804dbbeba9.jpg…]()

**Comment by Vikassm73** [comment]
Now, I am able to replicate this error. Let me check.

**Comment by SaiKiranCruzz** [comment]
Okay ty 😄

**Comment by SaiKiranCruzz** [comment]
If u can get us a AHA app add-on for kodi it will be amazing if u can do it Please take my request. it is also a ott app but for tollywood content

**Comment by Vikassm73** [comment]
zee5 new version update. Will you send kodi log after it works successfully?

**Comment by SaiKiranCruzz** [comment]
Yeah it is working just updated here is the log https://paste.kodi.tv/afehibixaw

**Comment by SaiKiranCruzz** [comment]
If u can get us a AHA app add-on for kodi it will be amazing if u can do it Please take my request. it is also a ott app but for tollywood content

**Comment by Vikassm73** [comment]
I may not be working on any new add-ons because of the challenging process of maintaining all of these.

**Comment by SaiKiranCruzz** [comment]
Okay no issues tysm

**Comment by SaiKiranCruzz** [comment]
If u have a yt channel or something lemme know im interested to learn these kind of stuff. I mean in making these addons n stuff ty

**Comment by Vikassm73** [comment]
If you have little bit programming knowledge, you can easily learn this. You need windows / Linux system. Install latest kodi on this. install add-on that and just go through the codes. Once you go through it, step by step you will understand the things.

**Comment by SaiKiranCruzz** [comment]
Okay 👍
